utils/vaultsigner/vaultsigner.go

```diff
@@ -159,3 +161,32 @@ func New(client *api.Client, name string) (*VaultSigner, error) {
 
 	return &VaultSigner{Client: client, KeyName: name, KeyVersion: 1}, nil
 }
+
+func Connect() (client *api.Client, err error) {
+	config := &api.Config{}
+	err = config.ReadEnvironment()
+
+	if err != nil {
+		client, err = api.NewClient(config)
+	} else {
+		client, err = api.NewClient(nil)
+		if err != nil {
+			log.Fatalln(err)
```

Comment:
we should probably just return the error here, better to be consistent with handling of other cases and allow the caller to decide correct behavior (esp since `log.Fatalln` calls `os.Exit(1)`)